DEVELOPMENT.md

```diff
@@ -114,6 +114,29 @@ We use [Cypress](https://www.cypress.io/) for integration tests and
 - To develop unit tests in watch mode, run: `yarn test:unit:watch`
 
 
+### Running multiple Upstream instances on the same machine
+
+At the moment this is only possible in development mode.
+
+```
+mkdir /Users/rudolfs/work/19000
+mkdir /Users/rudolfs/work/20000
+
+# Launch the first instance.
+
+RAD_HOME="/Users/rudolfs/work/19000" \
```

Comment:
Note you could also just use `RAD_PROFILE` which is what I did. Created two profiles and then started them with env var. Could be worth documenting here too

Reply:
f14408ec8d3bdf0f9d0dcb74c444a989c7b52414